src/FlipCounter.jsx

```diff
@@ -0,0 +1,74 @@
+import React, { Component } from "react";
+import "./FlipCounter.css";
+
+class Counter extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      x: 0
+  
+    };
+  }
+  add = (num) => {
+    this.setState({ x: parseInt(this.state.x) + 1 });
+    console.log("add activated");
```

Comment:
Need to remove all console.log

We never live them in the code, because each console.log affects perfomance of the code and create unnessary bloat in the developers console